utils/cautils/acmeutils.go

```diff
@@ -431,19 +430,35 @@ func (af *acmeFlow) GetCertificate() ([]*x509.Certificate, error) {
 		if err != nil {
 			return nil, err
 		}
+
+		// check if the list of identifiers for which to
+		// request a certificate already contains the subject
 		hasSubject := false
 		for _, n := range idents {
 			if n.Value == af.subject {
 				hasSubject = true
 			}
 		}
-		subjectIsNotAnIP := net.ParseIP(af.subject) == nil
-		if !hasSubject && subjectIsNotAnIP {
-			dnsNames = append(dnsNames, af.subject)
-			idents = append(idents, acme.Identifier{
-				Type:  "dns",
-				Value: af.subject,
-			})
+		// if the subject is not yet included in the slice
+		// of identifiers, it is added to either the DNS names
+		// or IP addresses slice and the corresponding type of
+		// identifier is added to the slice of identifers.
+		if !hasSubject {
+			ip := net.ParseIP(af.subject)
```

Comment:
Similar example of coding convention difference. In my opinion we should either write this as a one liner, or make an argument for why breaking it up is always better and start using the new convention everywhere. I'm gonna sound pedantic and whiny, but it makes the code harder to read when we're using two different conventions to handle the same situations throughout.